homeassistant/components/vicare/strings.json

```
@@ -10,13 +10,21 @@
           "client_id": "[%key:common::config_flow::data::api_key%]",
           "heating_type": "Heating type"
         }
+      },
+      "reauth_confirm": {
+        "description": "Please verify credentials.",
+        "data": {
+          "password": "[%key:common::config_flow::data::password%]",
+          "client_id": "[%key:common::config_flow::data::api_key%]"
```

Correct once #103111 is approved.

```suggestion
          "client_id": "[%key:component::vicare::config_flow::data::client_id%]"
```